commands/mr/mrutils/mrutils.go

```diff
@@ -151,6 +153,54 @@ func MRFromArgs(f *cmdutils.Factory, args []string) (*gitlab.MergeRequest, glrep
 	return mr, baseRepo, nil
 }
 
+func MRsFromArgs(f *cmdutils.Factory, args []string) ([]*gitlab.MergeRequest, glrepo.Interface, error) {
+	if len(args) <= 1 {
```

Comment:
There is a restriction on the `mr close` command on this line indicating it cannot have more than one positional arguments:
https://github.com/profclems/glab/blob/9b0868068e88acdec171aa07d6651df537b72986/commands/mr/close/mr_close.go#L21

I'm glad you did it this way so we can have room for cases where we allow multiple positional arguments (`glab mr close 1 2 3 4`) but for backward compatibility where the user can do `glab mr close 1,2,3,4` to close multiple MRs , you should check for when there is a single argument with comma-separated IDs.

```go
if len(args) <= 1 {
        var arrIds []string
        if(len(args) == 1) {
	        arrIds := strings.Split(mergeID, ",")
        }
        if (len(arrids) <= 1) {
		mrID, baseRepo, err := MRFromArgs(f, args)
		if err != nil {
			return nil, nil, err
		}
		return []*gitlab.MergeRequest{mrID}, baseRepo, err
	}
        args = arrIds
}
```

Reply:
I didn't think of the backwards compatibility with comma separated arguments. 
In this MR I ended up removing the constrain to the maximum arguments, but the correct thing would be to just revert back to the single argument and have it comma separated, and do it this way.

Reply:
Off-topic: Do you mind that I force push to my PRs? 
I know some people don't like when someone does that because they like to go through PRs commit by commit, but I like to have my commits tidy, and if I make a mistake in a commit I ammend to it and force push it, like the mistake never existed.

Reply:
That's okay

Reply:
You can merge this PR now. I have reviewed and approved